src/includes/getting-started-config/javascript.remix.mdx

```diff
@@ -0,0 +1,120 @@
+To use this SDK, initialize Sentry in your Remix entry points for both the client and server.
+
+```typescript {filename: entry.client.tsx}
+import { useLocation, useMatches } from "@remix-run/react";
+import * as Sentry from "@sentry/remix";
+import { useEffect } from "react";
+
+Sentry.init({
+  dsn: "__DSN__",
+  tracesSampleRate: 1,
+  integrations: [
+    new Sentry.BrowserTracing({
+        routingInstrumentation: Sentry.remixRouterInstrumentation(
```

Comment:
Is there a way for us to consume `useEffect`, `useLocation` & `useMatches` inside of `Sentry.remixRouterInstrumentation()`?

This way, we would provide a much better developer experience and we should have access to these APIs inside our function.

Reply:
AFAIK, this is the only way of passing those hooks. We're not dependent on either `remix` or `react` for the SDK, even if we did, we can't control the version clashes.

Developer experience for `react-router` v6 [is similar](https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/react-router/), so if there's an alternative way, we can fix both.